File: javascript/delete_inferenceclassificationoverride.js
var data = "{ }";

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("DELETE", "https://graph.microsoft.com/v1.0/me/inferenceClassification/overrides/98f5bdef-576a-404d-a2ea-07a3cf34af4r");
xhr.setRequestHeader("content-type", "application/json");

xhr.send(data);

File: java/delete_inferenceclassificationoverride.java
HttpResponse<String> response = Unirest.delete("https://graph.microsoft.com/v1.0/me/inferenceClassification/overrides/98f5bdef-576a-404d-a2ea-07a3cf34af4r")
  .header("content-type", "application/json")
  .body("{ }")
  .asString();
